doc/aurpublish.1.asciidoc

```diff
@@ -0,0 +1,83 @@
+aurpublish(1)
+=============
+
+Name
+----
+aurpublish - PKGBUILD management framework for the Arch User Repository
+
+
+Synopsis
+--------
+'aurpublish' [options] PACKAGE
+'aurpublish' log [<options>] PACKAGE
+'aurpublish' setup
```

Comment:
These need to be separated by one empty line each.

Reply:
Fixed in https://github.com/eli-schwartz/aurpublish/pull/16/commits/8e17120aff596e68474c2e29b554e3dc3173120c.